projects/file-picker/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Angular Library for uploading files with Real-Time Progress bar, File Preview, Drag && Drop and Custom Template with Multi Language support",
   "peerDependencies": {
     "@angular/common": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0",
-    "@angular/core": " ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0"
+    "@angular/core": " ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0",
+    "mrmime": "^1.0.0"
```

Comment:
1) Let's move this package to `"dependencies" `so users don't have to install this package manually.
2) Let's remove ^ caret to install safe and exact version.

Reply:
i pushed a commit with the required fixes